ex02.ex

```diff
@@ -53,11 +53,15 @@ defmodule Ex02 do
   # if the first two elements of a list are equal                #
   ################################################################
 
-  first2equal = your_anonymous_function([])
+  first2equal = fn 
+    [a, a | _] -> true
+    [_ | _] -> false
```

Comment:
### -1

This is a strange pattern, as it matches a list with 1 or more elements. Why would `first2equal[4]` return `false` but `first2equal([])` raise an exception?